app/assets/javascripts/charts.js

```diff
@@ -0,0 +1,21 @@
+$(function() {
+  $('.row a').on('click',  function(event)  {
+    event.preventDefault(); // Prevent link from following its href
+    var previous_poll_id = $('#previous-poll').data('poll_id');
+    var next_poll_id = $('#next-poll').data('poll_id');
+    var get_url;
+    if(this.id=="previous_graph") {
+      get_url = "api/polls/" + previous_poll_id;
+    }
+    else {
+      get_url = "api/polls/" + next_poll_id;
+    }
+
+    $.get(get_url, function(data){
+      data: $(event.target).serialize();
```

Comment:
This is not doing anything. It's actually being interpreted as a [label](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/label) which is why no syntax error is showing up.